examples/examples/rpc_middleware.rs

```diff
@@ -149,11 +203,11 @@ async fn run_server() -> anyhow::Result<SocketAddr> {
 	let global_cnt = Arc::new(AtomicUsize::new(0));
 
 	let rpc_middleware = RpcServiceBuilder::new()
```

Comment:
And then something like:

```rust
// We can also use the same middleware in the server, too..
```

To emphasise how we can now use the same middleware in both places